widget/embedded/src/pages/LiquiditySourcePage.tsx

```diff
@@ -145,14 +150,28 @@ export function LiquiditySourcePage({
           value={searchedFor}
         />
 
-        <ListContainer>
-          <List
-            type={
-              <ListItemButton title="_" id="_" onClick={() => console.log()} />
-            }
-            items={filteredList}
-          />
-        </ListContainer>
+        {!filteredList.length && !!searchedFor ? (
+          <>
+            <Divider size={32} />
+            <NotFound
```

Comment:
the place of NotFound is not matched with figma
![image](https://github.com/rango-exchange/rango-client/assets/123297941/9fbcb9dc-f44c-459c-b2bb-f3a167f8e3c9)

Reply:
Done